src/android/nl/xservices/plugins/Calendar.java

```diff
@@ -694,18 +696,21 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
       if (resultCode == Activity.RESULT_OK || resultCode == Activity.RESULT_CANCELED) {
         // resultCode may be 0 (RESULT_CANCELED) even when it was created, so passing nothing is the clearest option here
         Log.d(LOG_TAG, "onActivityResult resultcode: " + resultCode);
-        callback.success();
+        if (callback != null)
+          callback.success();
       } else {
         // odd case
         Log.d(LOG_TAG, "onActivityResult weird resultcode: " + resultCode);
-        callback.success();
+        if (callback != null)
+          callback.success();
       }
     } else if (requestCode == RESULT_CODE_OPENCAL) {
       Log.d(LOG_TAG, "onActivityResult requestCode: " + RESULT_CODE_OPENCAL);
-      callback.success();
+      // Callback is called right after startActivityForResult
     } else {
       Log.d(LOG_TAG, "onActivityResult error, resultcode: " + resultCode);
-      callback.error("Unable to add event (" + resultCode + ").");
+      if (callback != null)
+        callback.error("Unable to add event (" + resultCode + ").");
```

Comment:
If the activity got destroyed, methods that open native screens
(open calendar; create event with dialog) crash when calling the callback.

This happens because the callback reference no longer exists.
This change adds a null-check to said variable.